plugins/rapid7_insightidr/komand_rapid7_insightidr/util/endpoints.py

```diff
@@ -146,6 +146,17 @@ def add_indicators_to_a_threat(console_url: str, key: str):
 
         return f"{console_url}idr/v1/customthreats/key/{key}/indicators/add"
 
+    @staticmethod
+    def replace_indicators(console_url: str, key: str):
```

Comment:
I understand none of the rest of them have this included, maybe worth doing in another PR or including it in this one for this method and all other ones doing the same thing
```suggestion
    def replace_indicators(console_url: str, key: str) -> str:
```